### ISSUE TITLE
Self-Hosted Not Working For Some Time

### ISSUE THREAD

**Issue by elsiehupp** [forwardemail/free-email-forwarding#207]
I didn't notice until recently that I hadn't been getting emails from Amazon, which... having automated emails not forward correctly (since I give my real email address to real human beings) isn't the most annoying thing, but those emails are occasionally actually important.

I just sent myself a test email (to a different address on the same server), and it bounced with the error "550 queryTxt ETIMEOUT".

I also tried ismyemailworking.com/advancedtest, and I got a different 550 error at the following stage:

```
03/01/2020 10:01:34 - [Exception: 550 queryTxt ESERVFAIL elsiehupp.com]
03/01/2020 10:01:34 - ['rcpt to' transmission failed]
```

I set up several hundred as-yet-unused aliases in my TXT records a couple months ago, but it looks like I stopped receiving emails from Amazon even before that, back in November. I am having a difficult time pinning down the last forwarded email I received, but it looks like it was over a month ago.

My most plausible guess with the "ETIMEOUT" error is that it might be a result of having an unusually large number of TXT entries (literally 459 of them, apparently), though it's also possible that something else broke behind the scenes.

Again, the my forward-email instance ***was*** working, but then it stopped without any notice.

If you want to try to reproduce the error for yourself, you can send a test email to the address "test" at the domain name in the error message earlier in this post. (I'm not putting it in plain text here because I don't want it to be easy to scrape.)

**Comment by niftylettuce** [comment]
This is very vague, what commit are you working off of?  It seems obvious that the hundreds of records are the reason for error.

**Comment by elsiehupp** [comment]
Okay, to be less vague: I set up a DigitalOcean droplet following the instructions that were on forwardemail.net back in November. These instructions are no longer anywhere to be found, so I have no way to retrace my steps.

It looks like my Amazon email alias has not been working since even before #192. In other words, it never worked. I have done nothing since then other than adding additional TXT entries, which I did a month or two ago, without actually making any use of them. I don't know when my previously functional aliases stopped working, because they failed silently.

Regarding the "obviousness" of having hundreds of records itself being an error condition... the documentation in place last November made no mention of an upper limit on the number of TXT entries. The main reason I switched to forward-email in the first place was the lack of a stated upper limit in the number of aliases.

Again, in the meantime, you have completely redesigned your website to emphasize a completely different set of solutions, so I have no idea where to even look anymore to diagnose my problem. The vagueness of my issue seems to be directly related to the change in your business model, to the extent that your dismissiveness comes across as disingenuous. Or maybe you're just being subconsciously sexist. Either way, color me unimpressed. Maybe I should just be like #175, call you unreliable, and switch to a competitor. You're not doing a very good job of encouraging me to become a paying customer of your new, fully hosted service.
